pytim/__init__.py

```diff
@@ -323,11 +328,16 @@ def assign_alpha(self, alpha):
         self.interface.alpha = alpha
         return True
 
-    def assign_groups(self, itim_group, cluster_cut, extra_cluster_groups):
+    def assign_groups(self,universe, itim_group, cluster_cut, extra_cluster_groups):
```

Comment:
no need to pass `universe` here, it's supposed to be already defined as `self.universe`. Can enclose the relevant code in try/except to catch misuse of `assign_group` (e.g. if called before `assign_universe`